src/components/price/price.tsx

```diff
@@ -0,0 +1,16 @@
+import styles from './price.module.scss';
+
+export interface PriceProps {
+    fullPrice: string | undefined;
+    price: string | undefined;
+}
+
+export const Price = ({ price, fullPrice }: PriceProps) => {
+    return price ? (
+        <div className={styles.container}>
+            {fullPrice !== price && <div className={styles.fullPrice}>{fullPrice}</div>}
+
+            <div className={styles.currentPrice}>{price}</div>
+        </div>
+    ) : undefined;
```

Comment:
why does this component return `undefined`?

Reply:
to simplify its usage across project...
for some reasons, price from API might be undefined, so we have two options:

```
<Price
      fullPrice={price?.formatted?.price}
      price={price?.formatted?.discountedPrice}
  />
```
vs
```
price?.formatted?.price && <Price
                                                  fullPrice={price.formatted.price}
                                                  price={price.formatted.discountedPrice}
                                              />
```

Reply:
Didn't get which of those is simpler usage. (in my opinion the first one)
In current implementation if `props.price` is undefined but `props.fullPrice` - not, component will return `undefined`, right?

Reply:
correct, if there is no `currentPrice` (price) - it will return undefined

Reply:
I prefer second option, but I would return early if price is undefined.
to reduce indentation

Reply:
I believe the component interface should be
```
interface PriceProps {
    currentPrice: string;
    previousPrice?: string;
```

Reply:
fixed